micrometer-core/src/main/java/io/micrometer/core/instrument/binder/httpcomponents/PoolingNHttpClientConnectionManagerMetricsBinder.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2019 Pivotal Software, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.core.instrument.binder.httpcomponents;
+
+import io.micrometer.core.instrument.*;
+import io.micrometer.core.instrument.binder.MeterBinder;
+import io.micrometer.core.lang.NonNull;
+import org.apache.http.impl.nio.conn.PoolingNHttpClientConnectionManager;
+
+/**
+ * Collects metrics from a {@link PoolingNHttpClientConnectionManager}.
+ *
+ * It monitors the overall connection pool state.
+ *
+ * @author Daniel Albuquerque
+ * @since 1.3.2
```

Comment:
Since this is a new feature, it would be added to the next feature release, which is currently 1.4.0.